master/internal/api_user.go

```diff
@@ -204,6 +212,41 @@ func (a *apiServer) GetUser(
 	return &apiv1.GetUserResponse{User: toProtoUserFromFullUser(*targetFullUser)}, err
 }
 
+func (a *apiServer) GetMe(
+	ctx context.Context, req *apiv1.GetMeRequest,
+) (*apiv1.GetMeResponse, error) {
+	curUser, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+	curFullUser, err := getFullModelUser(curUser.ID)
+	if err != nil {
+		return nil, err
+	}
+	return &apiv1.GetMeResponse{User: toProtoUserFromFullUser(*curFullUser)}, err
+}
+
+func (a *apiServer) GetUserByUsername(
+	ctx context.Context, req *apiv1.GetUserByUsernameRequest,
+) (*apiv1.GetUserByUsernameResponse, error) {
+	curUser, _, err := grpcutil.GetUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+	targetFullUser, err := getFullModelUserByUsername(req.Username)
+	if err != nil {
+		return nil, err
+	}
+
+	var ok bool
+	if ok, err = user.AuthZProvider.Get().CanGetUser(ctx, *curUser, targetFullUser.ToUser()); err != nil {
+		return nil, err
+	} else if !ok {
+		return nil, errUserNotFound
+	}
```

Comment:
this is a nitpick but I believe more idiomatic style would be something like:

```suggestion
	ok, err := user.AuthZProvider.Get().CanGetUser(ctx, *curUser, targetFullUser.ToUser())
	if err != nil {
		return nil, err
	}
	if !ok {
		return nil, errUserNotFound
	}
```